core/src/main/java/org/frankframework/jta/narayana/NarayanaJtaTransactionManager.java

```diff
@@ -72,6 +73,8 @@ private void initialize() throws TransactionSystemException {
 
 			try {
 				arjPropertyManager.getCoreEnvironmentBean().setNodeIdentifier(getUid());
+
+				jtaPropertyManager.getJTAEnvironmentBean().setXaRecoveryNodes(List.of(getUid()));
```

Comment:
Willen we dit nog aan/uit kunnen zetten?

Reply:
Ik neem aan van die. Dit zorgt voor betere afhandeling van xids binnen de zelfde pod. In werkelijkheid als pods een andere naam krijgen zal dit niet helpen. Maar het is meer dan het was.